docs/faq.md

```diff
@@ -158,14 +158,29 @@ docker exec -t pmm-server bash -c  "grafana-cli --homepath /usr/share/grafana a
 
 If you're deploying through Docker you can change the default password for an admin user after starting the Docker container as follows:
 
-    * For PMM versions 2.27.0 and later:
+* For PMM versions 2.27.0 and later:
 
-    ```sh
-    docker exec -t pmm-server change-admin-password <new_password>
-    ```
+```sh
+docker exec -t pmm-server change-admin-password <new_password>
+```
+
+* For PMM versions prior to 2.27.0:
+
+```sh
+docker exec -t pmm-server bash -c 'grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password newpass'
+```
+
+## How to use a non-default listen-port for pmm-admin?
 
-    * For PMM versions prior to 2.27.0:
+Currently, if you configure `pmm-agent` to use a port other than the default port (7777), `pmm-admin` does not use this value from the configuration file `/usr/local/percona/pmm2/config/pmm-agent.yml` and tries to connect to the default port. The workaround is to use the following global flag to use a non-default listen-port:
```

Comment:
Maybe we can call it like new feature not as a workaround. @denisok What do you think?

Reply:
hm, this definitely not a workaround. It is not a new feature either. I would just say that: if Agent is configured for none default port pmm-admin should know that to communicate with the agent and thus there is a new parameter for it.